doc/source/getting_started/compatibility.rst

```diff
@@ -59,4 +52,4 @@ libraries.
      - 0.2.*
      - 0.1.0
 
-(** Compatibility of DPF 2.0 with ansys-dpf-core 0.5.0 to 0.9.0 is assumed but not certified.)
+** Compatibility of DPF 2.0 with PyDPF-Core 0.5.0 to 0.9.0 is assumed but not certified.
```

Comment:
The table uses both a single asterisk and a double asterisk. However, there is only a footnote for the double asterisk. What does the single asterisk mean in line 59?

Reply:
That is a very good question, I'll need to check in the history of modifications.

Reply:
@Paul--I changed the entry from ``0.2.*`` to ``2.0 to 2.x`` and removed the single asterisk. Then, I used the single asterisk for what is the only true table footnote.